File: AD-HOC/2229.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(){
    double t = 1, aplicacoes = 0, produzidos = 0;
    scanf("%lf", &aplicacoes);

    while(aplicacoes != -1){
        printf("Teste %.0lf\n", t);
        t += 1;
        produzidos = (1 + pow(2, aplicacoes)) * (1 + pow(2, aplicacoes));
        printf("%.0lf\n\n", produzidos);
        scanf("%lf", &aplicacoes);
    }
    return 0;
}

File: AD-HOC/2229.py
#Ler teste
t = 1
aplicacoes = int(input())
#Enquanto a leitura for diferente de -1...
while aplicacoes != -1:
    #Escrever id dos casos de teste
    print('Teste %d'%(t))
    t += 1
    #computar a quantidade de recortes
    produzidos = (1 + 2**aplicacoes)*(1 + 2**aplicacoes) #(1 + 2**aplicacoes)**2
    print(produzidos)
    print()
    aplicacoes = int(input())
